content/blog/2020-01-29-winter-school-recap.md

```diff
@@ -0,0 +1,35 @@
+
+Our goal wasn't to repeat the same material covered in bootcamps and Software Carpentry style workshops, but to focus on the research software engineering skills that are not formally taught in any setting. These skills include best practices for packaging code as software, testing, collaborative software development, code review, and related topics such as licensing and archiving.  Two months before the workshop, we asked each student to select a software project related to their research to bring to the workshop. Our intent was to intermingle instructions with work time for students to apply these skills to their own projects.
+
+
+### **Lessons learned**
+
+If anything it has become very clear to us that there is a strong demand for these skills, based on the demand for participation and general positive feedback from participants (more on that). We also found that, in this pilot, lectures and discussion ended up filling most of the time. We had originally scheduled roughly the same time for working on projects, but used much of that time for extended discussions. In the future we would like to run a longer workshop to allow students more time to work on their projects and apply the lessons they have learned in practice.
+
+We also found that it was really important to have supplemental instructors (“helpers”), beyond the three lead organizers, who could help teach some material and/or answer specific questions. Through some advertising and direct requests, we found six teaching assistants from diverse backgrounds who brought a wealth of technical and domain knowledge.
+
+
+### **Feedback**
+
+Despite our modest expectations for a first run of the course, the response from the participants was overwhelmingly positive. Here’s what a few participants had to say about the workshop:
+
+ \
+–”I really can't say enough good things about this super empowering workshop! You did an amazing job identifying the things I didn't know that I didn't know, and teaching them at a level that was immediately actionable in my work.”
+
+ – “Thank you so much to everyone for taking this amazing initiative to teach young scientists on software sustainability.”
+
+ – “Thank you for putting together this winter-school, it was super useful to me and I'm looking forward to applying everything I learned to my future projects and to go deeper into the topics that were covered.”
+
+We look forward to teaching a longer version of this school once the full institute is conceived.
```

Comment:
maybe end with a different work - conceived seems too similar to conceptualized - initiated? funded? starts?